src/python-common/ceph/deployment/drive_group.py

```diff
@@ -45,7 +45,16 @@ def __init__(self,
         self.actuators = actuators
 
         #: List of Device objects for devices paths.
-        self.paths = [] if paths is None else [Device(path) for path in paths]  # type: List[Device]
+
+        self.paths = []
+
+        if paths is not None:
+            for device in paths:
+                if isinstance(device, dict):
+                    path: str = device.get("data", '')
```

Comment:
NIT, what do you think about just keeping the name `path` here. The upper level is the plural `paths` so there would be no conflict and it's more intuitive to me than `data` as an option to specify a device path.

Reply:
Of course, having `path` instead of `data` makes sense and it's probably more consistent with the current implementation: I'll change this part in the next iteration and update the doc change as well.